quaint/src/visitor/postgres.rs

```diff
@@ -499,6 +500,45 @@ impl<'a> Visitor<'a> for Postgres<'a> {
         }
     }
 
+    #[cfg(any(feature = "postgresql", feature = "mysql"))]
+    fn visit_json_array_agg(&mut self, array_agg: JsonArrayAgg<'a>) -> visitor::Result {
+        self.write("JSONB_AGG")?;
+        self.surround_with("(", ")", |s| s.visit_expression(*array_agg.expr))?;
+
+        Ok(())
+    }
+
+    #[cfg(any(feature = "postgresql", feature = "mysql"))]
+    fn visit_json_build_object(&mut self, build_obj: JsonBuildObject<'a>) -> visitor::Result {
+        const MAX_FIELDS: usize = 50;
+        let num_chunks = build_obj.exprs.len().div_ceil(MAX_FIELDS);
+
+        for (i, chunk) in build_obj.exprs.into_iter().chunks(50).into_iter().enumerate() {
+            let mut chunk = chunk.peekable();
+
+            self.write("JSONB_BUILD_OBJECT")?;
+
+            self.surround_with("(", ")", |s| {
+                while let Some((name, expr)) = chunk.next() {
+                    s.visit_raw_value(Value::text(name))?;
+                    s.write(", ")?;
+                    s.visit_expression(expr)?;
+                    if chunk.peek().is_some() {
+                        s.write(", ")?;
+                    }
+                }
+
+                Ok(())
+            })?;
+
+            if i < num_chunks - 1 {
+                self.write("|| ")?;
+            }
```

Comment:
Could you also write a test, at the bottom of this file, to assert that the generated query is being chunked?